dso-l2/src/main/java/com/tc/l2/state/ConsistencyManager.java

```diff
@@ -53,7 +55,11 @@ boolean isStateTransition() {
     }
   }
 
-  boolean requestTransition(ServerMode mode, NodeID sourceNode, Transition newMode) throws IllegalStateException;
+  default boolean requestTransition(ServerMode mode, NodeID sourceNode, Transition newMode) throws IllegalStateException {
+    return requestTransition(mode, sourceNode, TopologyManager.get().getTopology(), newMode);
```

Comment:
maybe get rid of the singleton and pass null?  use legacy behavior when null is passed.

Reply:
done